dashactivity.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * 2007-2016 PrestaShop
+ * 2007-2018 PrestaShop
```

Comment:
2019?

Reply:
ping @matks

Reply:
I'm sorry but I submitted this PR in 2018 so license headers were fine at this time 😅. Now even Readme.md is in conflict.

It's hard having to come back to this PR 2 years after because of an issue that did not exist at the time I submitted the PR 😢